hardware/asic/skywater/skywater130.mk

```diff
@@ -1 +1,7 @@
 #Makefile for OpenLane and OpenRAM started on 2/11/21
+SRAM:
+	python3 $(OPENRAM_HOME)/openram.py $(OPENRAM_CONFIG)/myconfig.py
```

Comment:
OPENRAM_HOME should be OPENRAM_COMPILER
OPENRAM_CONFIG should be OPENRAM_HOME

Reply:
OPENRAM_HOME is py file for OpenRAM
and OPENRAM_CONFIG has the configuration that user can modify to change all the params of the SRAM
I will modify them now